controllers/authpolicy_istio_authorization_policy.go

```diff
@@ -6,6 +6,7 @@ import (
 	"reflect"
 
 	"github.com/go-logr/logr"
+	"golang.org/x/exp/slices"
```

Comment:
This slice package (https://pkg.go.dev/golang.org/x/exp@v0.0.0-20230817173708-d852ddb80c63/slices) is provided as part of Go 1.21 standard library https://pkg.go.dev/slices@go1.21.0 but is not available in Go 1.20